src/services/db/CommitServiceGithub.ts

```diff
@@ -0,0 +1,80 @@
+import { GrowthBook } from "@growthbook/growthbook"
+import { AxiosCacheInstance } from "axios-cache-interceptor"
+
+import UserWithSiteSessionData from "@root/classes/UserWithSiteSessionData"
+import { FEATURE_FLAGS } from "@root/constants"
+import { FeatureFlags } from "@root/types/featureFlags"
+import { isReduceBuildTimesWhitelistedRepo } from "@root/utils/growthbook-utils"
```

Comment:
casual reminder to remove unused vars

Reply:
59732ce1515f2bbf561a3efdf4b9e6107e98e62f